setup.py

```diff
@@ -83,7 +83,7 @@
     ],
     install_requires=[
         "django-appconf>=1.0.1",
-        "jsonfield>=1.0.3",
+        "jsonfield~=1.0.3",
```

Comment:
Oh, oops -- this is the wrong syntax -- I'm limiting to `1.0.x` here...

I'll rebase